terraform/main.tf

```diff
@@ -625,6 +625,8 @@ module "compute" {
   keystore_password_id      = local.keystore_password_id
   root_ca_id                = local.root_ca_id
   thread_pool_limit         = var.thread_pool_limit
+  wls_admin_user_1          = var.wls_admin_user_1
```

Comment:
Continuing from my comment on the image MR 1022 , I prefer the naming to be wls_secondary_admin for these. Minor comment. Let me know what you think.